src/components/app.jsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import LocationDetails from './location-details';
+import ForecastSummaries from './forecast-summaries';
+import ForecastDetails from './forecast-details';
+import axios from 'axios';
+import SearchForm from './search-form';
+import '../styles/app.scss';
+import '../styles/forecast-summary.scss';
+import '../styles/forecast-summaries.scss';
+import '../styles/search-form.scss';
+import '../styles/forecast-details.scss';
+
+const URL = 'https://mcr-codes-weather.herokuapp.com/forecast?city=';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      searchText: '',
+      searchError: '',
+      selectedDate: 0,
+      forecasts: [],
+      location: {
+        city: '',
+        country: '',
+      },
+    };
+  }
+
+  componentDidMount() {
+    this.locationSearch('manchester');
+  }
+
+  handleForecastSelect = (date) => {
+    this.setState({
+      selectedDate: date,
+    });
+  };
+
+  handleInputChange = (event) => {
+    this.setState({ searchText: event.target.value });
+    if (event.target.value === '') {
+      this.setState({ searchError: '' });
+    }
+  };
+
```

Comment:
Good idea putting in this event handler. Most people take it for granted that hitting enter will send the form. If it doesn't then I can be very irritating.